src/utils/date.ts

```diff
@@ -1,13 +1,89 @@
-const getDateDisplayValue = (date: string | number) => new Date(date).toISOString().split('T')[0];
+import { DateTime } from 'luxon';
 
-export const getTodaysDateFormatted = () => {
-  return getDateDisplayValue(Date.now());
+/**
+ * Helper util that returns default time zone as a fallback.
+ */
+const tz = (timeZoneId?: string) => timeZoneId || 'Etc/UTC';
+
+/**
+ * Parse a date string with zone implied.
+ * eg. '2023-01-10 in 'America/Los_Angeles'
+ */
+const getDateFromString = (date: any, timeZoneId?: string) => {
+  return DateTime.fromFormat(`${date} ${tz(timeZoneId)}`, 'yyyy-MM-dd z');
+};
+
+/**
+ * Parse a date from milliseconds, with time zone implied.
+ */
+const getDateFromMillis = (date:any , timeZoneId?: string) => {
+  const millis = new Date(date).getTime();
+
+  return DateTime.fromMillis(millis, { zone: tz(timeZoneId) });
+};
+
+/**
+ * Return the display value for a date in ISO 8601 format.
+ */
+const getDisplayValue = (date: any) => {
```

Comment:
Is `any` the correct type here? Or `Date` or `DateTime`?

Reply:
this could be DateTime, let me check

Reply:
yup DateTime, will update.. and the others are string | number